### PR TITLE
Batch operation of devices

### LINKED ISSUE #115: Batch operation of devices
Implementation of batch operation for devices app
| Short Name           | Descriptions                                                                                                                                                       | Short-Term | Long-Term |
|----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------|-----------|
| Batch Create         | User can create multiple devices at the same time.                                                                                                                 |     |           |
| Batch Delete         | User can delete the devices matching the filter, or delete multiple selected devices.

### PR DESCRIPTION
**Pull request by djs0109**
close #115

### CODE REVIEW

**Comment by djs0109** [comment]
@sbanoeon @SBlechmann Do you think the batch creation of devices is necessary? For me, if the user want to provision a series of devices, he/she should just create a service group and activate the `autoprovision`. However, if `autoprovision` is absolutely not wanted, I can think of creating series of devices based on a service group. What do you think?

File: app/Entirety/devices/templates/devices/list.html [hunk 1]
```diff
@@ -69,7 +69,7 @@ <h3>Devices</h3>
             </button>
          </div>
 
-            {% render_table table_devices %}
+            {% render_table tables.0 %}
```
Comment (sbanoeon) [inline app/Entirety/devices/templates/devices/list.html:76]: Can you please limit number of characters to show on list view for Device Id. Currently, if the Device Id is too long, the format of table breaks.

**Comment by sbanoeon** [comment]
In the create view for Devices, I see that there are no suggestions for Entity Type, is it intended this way?

**Comment by SBlechmann** [comment]
> @sbanoeon @SBlechmann Do you think the batch creation of devices is necessary? For me, if the user want to provision a series of devices, he/she should just create a service group and activate the `autoprovision`. However, if `autoprovision` is absolutely not wanted, I can think of creating series of devices based on a service group. What do you think?

Imo, we should provide that feature, yes. Autoprovisioning is an optional feature so we need to provide alternatives.

**Comment by djs0109** [comment]
@sbanoeon can you review it again? The point of device id length limitation will still be updated

File: app/Entirety/devices/templates/devices/list.html (hunk 1 above)
Comment (djs0109) [reply app/Entirety/devices/templates/devices/list.html:76]: Like in the project app? Do you already have an idea?

Comment (djs0109) [reply app/Entirety/devices/templates/devices/list.html:76]: I found a solution and also implemented it for entities app. What do you think?

**Review by sbanoeon — APPROVED** [review]

**Pull request merged**